packages/fluent-editor/src/modules/link/modules/tooltip.ts

```
@@ -312,6 +313,8 @@ export class LinkTooltip extends BaseTooltip {
 
   // @ts-expect-error
   edit(mode: string = 'link', preview = null, range) {
+    console.log('1111 edit')
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)